### PR TITLE
exclude jackson-datatype-jsr310 when included debezium

### PR DESCRIPTION
**Pull request by mogoodrich**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -111,6 +111,10 @@
 					<groupId>com.fasterxml.jackson.core</groupId>
 					<artifactId>jackson-annotations</artifactId>
 				</exclusion>
+				<exclusion>
+					<groupId>com.fasterxml.jackson.datatype</groupId>
+					<artifactId>jackson-datatype-jsr310</artifactId>
+				</exclusion>
```
Comment (mogoodrich) [inline pom.xml:117]: This was the the only com.fasterxml.jackson library included by core that was not excluded here.

**Review by mogoodrich — COMMENTED** [review]
With this addition, OpenMRS 2.6.1 starts up with the DB Event module.

I only quickly tested, and haven't done any testing to confirm the DB Event module works, but wanted to get this up for review, can test more next week before merging. @mseaton

**Review by mseaton — APPROVED** [review]
LGTM, thanks for figuring this out @mogoodrich .  I don't expect any problems on merge.

**Comment by mogoodrich** [comment]
Merging this in, thanks @mseaton !

**Pull request merged**